draft-ietf-acme-acme.md

```diff
@@ -1641,6 +1654,15 @@ To request authorization for an identifier, the client sends a POST request to
 the new-authorization resource specifying the identifier for which authorization
 is being requested.
 
+Any identifier of type "dns" in a pre-authorization request MAY have a wildcard
```

Comment:
Let's please change this `MAY` to `MUST NOT` and clean up the text that follows.  Preauth is for authorizing specific identifiers; each authz corresponds to a single identifier, so the identifier in the request should equal the one in the authz.  With wildcards, there's ambiguity about what identifier will be in the authz, or whether multiple authzs might be needed.  Let's just avoid that complexity.

Reply:
> Let's please change this MAY to MUST NOT

Sure, sounds good.

>  Preauth is for authorizing specific identifiers;

I don't think this is clear from the text that exists today. I'll remove everything but the "Wildcard" authz field text from this PR since this is feedback for https://github.com/ietf-wg-acme/acme/pull/401 but I [stand by my comment there](https://github.com/ietf-wg-acme/acme/pull/401#issuecomment-370148270): You and EKR have a notion of what pre-authorization can be used for that isn't reflected in the text. There should be a PR and a mailing list discussion about introducing text that clarifies the pre-authorization restrictions compared to newOrder.